pyvista/core/filters/poly_data.py

```diff
@@ -1700,6 +1710,16 @@ def compute_normals(
         _update_alg(normal, progress_bar, 'Computing Normals')
 
         mesh = _get_output(normal)
+        try:
+            mesh['Normals']
+        except KeyError:
+            if (self.n_verts + self.n_lines) == self.n_cells:
+                raise TypeError(
+                    "Normals cannot be computed for PolyData containing only vertex cells (e.g. point clouds)\n"
+                    "and/or line cells. The PolyData cells must be polygons (e.g. triangle cells)."
+                )
+            else:
+                raise RuntimeError('Normals could not be computed.')
```

Comment:
I cannot assume that this may occur, but if it does, we want to have the issue reported.
```suggestion
            else:
                raise RuntimeError(
                    'Normals could not be computed for unknown reasons.\n'
                    'Please report the issue at https://github.com/pyvista/pyvista/issues.\n'
                )
```